features/mbedtls/README.md

```diff
@@ -1,14 +1,96 @@
-README for mbed TLS
+README for Mbed TLS
 ===================
 
-mbed TLS for mbed OS
+Mbed TLS for Mbed OS
 --------------------
 
-This edition of mbed TLS has been adapted for mbed OS and imported from its standalone release, which you can find on [github here](https://github.com/ARMmbed/mbedtls). This edition of mbed TLS does not include test code, sample applications, or the scripts used in the development of the library. All of these can be found in the standalone release.
+This edition of Mbed TLS has been adapted for Mbed OS and imported from its standalone release, which you can find on [github here](https://github.com/ARMmbed/mbedtls). This edition of Mbed TLS does not include the test code or the scripts used in the development of the library. All of these can be found in the standalone release.
 
 
-Getting Help and Support
-------------------------
+Getting Started
+---------------
+
+Several example programs are available that demonstrate the use of Mbed TLS with
+Mbed OS. These are a great way of getting to know the library.
+
+1. [**TLS Client:**](https://github.com/ARMmbed/mbed-os-example-tls/tree/master/tls-client) TLS Client demonstrates the use of Mbed TLS to establish a client TLS connection to a remote server.
+
+2. [**Benchmark:**](https://github.com/ARMmbed/mbed-os-example-tls/tree/master/benchmark) Benchmark measures the time taken to perform basic cryptographic functions used in the library.
+
+3. [**Hashing:**](https://github.com/ARMmbed/mbed-os-example-tls/tree/master/hashing) Hashing demonstrates the various APIs for computing hashes of data (also known as message digests) with SHA-256.
+
+4. [**Authenticated encryption:**](https://github.com/ARMmbed/mbed-os-example-tls/tree/master/authcrypt) Authcrypt demonstrates usage of the Cipher API for encrypting and authenticating data with AES-CCM.
```

Comment:
Why is ”Cipher API“ capitalized?

Reply:
Good question! I copied and pasted this from @mpg's original Yotta Readme. It looks wrong so I've corrected it.